here-naksha-lib-model/src/commonMain/kotlin/naksha/model/NakshaFeatureProxy.kt

```diff
@@ -0,0 +1,28 @@
+@file:Suppress("OPT_IN_USAGE")
+
+package naksha.model
+
+import naksha.base.NullableProperty
+import kotlin.js.JsExport
+import kotlin.jvm.JvmStatic
+
+/**
+ * The Naksha Feature extending the default [GeoFeature].
+ */
+@JsExport
+open class NakshaFeatureProxy : com.here.naksha.lib.base.GeoFeature() {
+
+    companion object {
+        @JvmStatic
+        val REFERENCE_POINT =  NullableProperty<Any, NakshaFeatureProxy, PointProxy>(
+            PointProxy::class)
```

Comment:
Can we simply make this private val, then we do not need the `@JvmStatic` and the Kotlin compiler can inline the getter/setter.